src/runtime/manifest.ts

```diff
@@ -361,17 +365,22 @@ export class Manifest {
         }
         console.warn(processError(error).message);
       }
+      if (options.logWarnings) {
+        for (const warning of manifest.warnings) {
```

Comment:
Personally, I'd much prefer to have the warnings print intermixed with other errors. Can we regain this by having a list of errors which each have an error level or something similar? It helps me to understand how the compiler/parser found the problems that it is telling me about.